CHANGES.rst

```diff
@@ -1,6 +1,8 @@
 platformdirs Changelog
 ======================
 
+- **BREAKING** Correct the log directory on Linux/Unix
```

Comment:
You need the version and release date header

Reply:
Should also mention from what we change to what.

Reply:
> You need the version and release date header

Why am I deciding what version this will be? and why am I determining the date that this will be released? I don't intend to become a maintainer of this project just because I opened a pull request.